### PR TITLE
Fix some bugs :bug:

### LINKED ISSUE #21: getShortcut will send errors if we ask for an unknow user
https://github.com/CouturierTristan/fast-react-bot/blob/9081f39a75129fd9cb615269f8b205ee6e8966cb/lib/shortcuts.js#L27

### PR DESCRIPTION
**Pull request by DoBrcd**
Solve #21

### CODE REVIEW

File: lib/shortcuts.js [hunk 1]
```diff
@@ -21,11 +21,16 @@ const addShortcut = (userId, name, emojis) => {
...
+  const user = db.get('users').find({ userId });
+  if (!user.value()) {
+    throw new Error(`${userId} don't have any shortcuts`);
+  }
+  return user.get('shortcuts').value();
 };
 
 const getShortcut = (userId, shortcutName) => {
-  return getAllByUser(userId).find((c) => c.name === shortcutName);
+  const shortcuts = getAllByUser(userId);
```
Comment (Ailurus13) [inline lib/shortcuts.js:42]: Better use `getShortcut` from shortcuts.js

Comment (DoBrcd) [reply lib/shortcuts.js:42]: https://github.com/CouturierTristan/fast-react-bot/pull/22/commits/e8d30ad48af3469f3d0ae4ff28e03ce5dd5b2797
Look at this commit

Comment (Ailurus13) [reply lib/shortcuts.js:42]: We can now delete the file `default-react.js`

**Review by Ailurus13 — CHANGES_REQUESTED** [review]

File: lib/shortcuts.js (hunk 1 above)
Comment (DoBrcd) [reply lib/shortcuts.js:42]: Done

**Pull request merged**